src/qt/uicomponents/ApplicationConfiguration.cpp

```diff
@@ -1,7 +1,99 @@
+#include <algorithm>
 #include <QFontDatabase>
 #include <QGuiApplication>
+#include <QMap>
+#include <QString>
+#include <QVariant>
 #include "ApplicationConfiguration.h"
 
+namespace
```

Comment:
I would much prefer to see themes loaded from the QML, not from C++. We want custom themes, so this must be supported anyway.

Reply:
Done.

Reply:
Is there a reason not to use [QPalette](https://doc.qt.io/qt-6.2/qpalette.html#details)?